editor/components/block-settings-menu/block-transformations.js

```diff
@@ -25,7 +25,7 @@ function BlockTransformations( { blocks, small = false, onTransform, onClick = n
 		return null;
 	}
 	return (
-		<div className="editor-block-settings-menu__section">
+		<Fragment>
```

Comment:
Minor: Is there a special reason for having a fragment here and not use array return?

Reply:
I prefer Fragment to avoid having to add keys and commas :)

Reply:
Ok it makes sense :)

Reply:
Oh I guess, your right here, since it's a just a map inside, we could drop it entirely 👍

Reply:
Initially, the separator was here :)